### ISSUE TITLE
collabora 4

### ISSUE THREAD

**Issue by dieg0nob** [husisusi/officeonlin-install.sh#175]

**Comment by dieg0nob** [comment]
Has someone been able to compile some version of loolwsd 4.X with libreoffice core 6.x?
branchs and tags in officeonlin-install.cfg?

**Comment by Ark74** [comment]
Seems, and it is only a lucky guess, that they'll move out of libreoffice as core, and set a new branch for code4 as core.

**Comment by Gomez** [comment]
Works here on Debian Stretch:
![online_office_6](https://user-images.githubusercontent.com/48724/56185490-a3352d00-601d-11e9-8d5c-f6225453326d.png)

I installed npm and used this branch: https://github.com/husisusi/officeonlin-install.sh/pull/178

**Comment by Gomez** [comment]
Issue can be closed.

**Comment by joergmschulz** [comment]
4.1.0 (git hash: 4185361e) 

Collabora OfficeDev 6.0-29 (git hash: 41d57bd)

work as well.

**Comment by Ark74** [comment]
4.0.4
6.0-30
:+1:

**Comment by joergmschulz** [comment]
this is, indeed, the most stable combination. 4.0.5 has an issue when you right-click (spellcheck doesn't open, rendering interrupted). But, files containing an % in filename can't be opened in 4.0.4 which is fixed in 4.0.5. Who has a better combo?

**Comment by Ark74** [comment]
I can't reproduce your issue with 
```
LOOLWSD
4.0.5 (git hash: 1e4154c)
LOKit
Collabora OfficeDev 6.0-31 (git hash: 9e571fc)
```
